cmd/infra/aws/create.go

```diff
@@ -184,7 +185,11 @@ func (o *CreateInfraOptions) CreateInfra(ctx context.Context) (*CreateInfraOutpu
 	if err != nil {
 		return nil, err
 	}
-	result.PrivateZoneID, err = o.CreatePrivateZone(ctx, route53Client, result.VPCID)
+	result.PrivateZoneID, err = o.CreatePrivateZone(ctx, route53Client, fmt.Sprintf("%s.%s", o.Name, o.BaseDomain), result.VPCID)
+	if err != nil {
+		return nil, err
+	}
+	_, err = o.CreatePrivateZone(ctx, route53Client, fmt.Sprintf("%s.%s", o.Name, hypershiftLocalZoneName), result.VPCID)
```

Comment:
do we need to create this one for non private topologies?

Reply:
we don't, but we don't know at this point whether the HC with be private or not as there is no flag to pass this information